__tests__/app.test.js

```diff
@@ -256,3 +256,19 @@ describe("DELETE /api/comments/:comment_id", () => {
     expect(res.body.msg).toBe("Bad Request. Invalid Input Type");
   });
 });
+
+describe("GET /api/users", () => {
+  test("Returns an array of user objects with properties username, name and avatar url", async () => {
+    const res = await request(app).get("/api/users").expect(200);
+    const users = res.body.users;
+    users.forEach((user) => {
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)